### PR TITLE
[2.0.x] Add pin configuration for Fysetc F6

### PR DESCRIPTION
**Pull request by mroote**
Adds pin config files for the Fysetc F6 controller. I've checked most of the pins on my own board and have fixed a few others from the schematic https://github.com/FYSETC/F6/blob/master/F6_V13.pdf.

Let me know if I'm missing adding something with the 2.0 code, I'm not running 2.0 yet on my printer.

### CODE REVIEW

File: Marlin/src/core/boards.h [hunk 1]
```diff
@@ -83,6 +83,7 @@
 #define BOARD_TRIGORILLA_13     343   // TriGorilla Anycubic version 1.3 based on RAMPS EFB
 #define BOARD_TRIGORILLA_14     443   // TriGorilla Anycubic version 1.4 based on RAMPS EFB
 #define BOARD_RAMPS_ENDER_4     243   // Creality: Ender-4, CR-8
+#define BOARD_FYSETC_F6_13      541   // Fysetc F6
```
Comment (thinkyhead) [inline Marlin/src/core/boards.h:87]: It's "**Fystec**" not "**Fysetc**."

Comment (mroote) [reply Marlin/src/core/boards.h:87]: No it is 'Fysetc', I thought the same until I double checked.

Comment (mroote) [reply Marlin/src/core/boards.h]: https://www.aliexpress.com/store/product/FYSETC-F6-Board-ALL-in-one-Electronics-Solution-For-3D-Printer-CNC-Devices-Up-to-6/3480083_32911155606.html?spm=2114.search0104.3.2.667d2d3bAGaIXK&ws_ab_test=searchweb0_0,searchweb201602_4_10065_10068_10130_10547_10059_10884_10548_10887_10696_100031_10084_10083_10103_10618_10139_10307,searchweb201603_2,ppcSwitch_0&algo_expid=1ad18130-78b6-4df3-ba90-91b849e144cc-0&algo_pvid=1ad18130-78b6-4df3-ba90-91b849e144cc&priceBeautifyAB=0

Comment (thinkyhead) [reply Marlin/src/core/boards.h:87]: OMG you're right. There's also a company called FYSTEC. They must be very annoyed.

**Pull request merged**